### DISCUSSION TITLE
How does this work

### DISCUSSION THREAD

**Discussion by Tonkikongo** [Aizistral-Studios/Enigmatic-Legacy#131] [Q&A] [answered]
Hi everyone, i play this great mod and i have some question to it, i dont find the answers in the mod or in the www i would be very grateful if someone could help me.

1. There are so many chapter blocked in book "The Arcknowledgment"  -> "Miscellaneous" , how can i activate them?

2. "Guide to Feral Hunt" &  "Guide to Animal Companionship"
Can i summon creatures the was fight for me? How does this work?

3. The drop chance of some items and ingredients is very low, can you increase it

4. Some items like "Heart of the Guardian" & "Heart of Creation" cannot be found in the open World / survival mode?

5. Someone is working on a translation into german language? :)

thx :)

**Comment by Aizistral** [comment, marked as answer]
1. Most entries in The Acknowledgement are unlocked when you either unlock recipe of item that entry is dedicated to in the recipe book, or, with items that cannot be crafted and are obtained otherwise, like from dungeon loot for instance - when you obtain such item itself. Collect ingredients, craft new items, explore the world - and greater knowledge will reveal itself to you.
2. Effects of those two items apply to any of your pets, like tamed dogs, cats or parrots.
3. More detail please, how can I increase drop chances if you don't tell what specific items have them too low?
4. Heart of Creation indeed cannot be found in survival mode, as it is designed to be a creative tool. Heart of the Guardian can be obtained as rare drop from Elder Guardian if you happen to wear Ring of the Seven Curses.
5. So far nobody came forth and offered their help with german localization, unfortunately. Would sure be great if someone in future.

**Reply by Tonkikongo** [reply]
Dear Extregral, first of all thank you for your quick and helpful answers.  

In addition to number 3: 
I cleared several fortresses, mines and other things but only very rarely got an item for example "spellstones" from the mod from boxes.

As for the translation, I can do that.

**Reply by Aizistral** [reply]
Spellstones are supposed to be rare, as much as the task of collecting them all - horrendously difficult. This is fair for how heavily stacked they are in traits and abilities they provide. The only other crucial item Enigmatic Legacy adds to dungeon loot of Overworld/Nether is Heart of the Earth, which is somewhat rare, but can be found in most types of dungeon chests in the Overworld, and you probably won't need many of those.

As for the translation, that would be nice of you. Are you aware already how translations are made and added to mods? I can familiarize you with the basics if you don't know what to begin with.
